test/functionalTest/harnessFunctions.sh

```diff
@@ -595,22 +625,200 @@ function dbInsertEntity()
 #   --host        <host>           (default: localhost)
 #   --port        <port>           (default: $CB_PORT)
 #   --url         <URL>            (default: empty string)
+#   --urlParams   <params>         (URI parameters 'in' URL-string)
```

Comment:
You have defined this new parameter for orionCurl but it seems that there isn't any .test using it?